mnist/README.md

```diff
@@ -1,170 +1,99 @@
-# Training MNIST using Kubeflow, S3, and Argo.
+# Training MNIST using Kubeflow, S3(Minio), and Argo.
 
-This example guides you through the process of taking an example model, modifying it to run better within Kubeflow, and serving the resulting trained model. We will be using Argo to manage the workflow, Tensorflow's S3 support for saving model training info, Tensorboard to visualize the training, and Kubeflow to deploy the Tensorflow operator and serve the model.
+This example guides you through the process of taking an example model, modifying it to run better within Kubeflow, and serving the resulting trained model. We will be using Argo to manage the workflow, Tensorflow's S3(Minio) support for saving model training info, Tensorboard to visualize the training, and Kubeflow to deploy the Tensorflow operator and serve the model.
 
 ## Prerequisites
 
 Before we get started there a few requirements.
 
-### Kubernetes Cluster Environment
+## 1.Kubernetes Cluster Environment
 
-Your cluster must:
+* Kubernetes v1.9.1+
```

Comment:
Can we just link to the Kubeflow getting started guide rather than duplicating the instructions here?

Reply:
yes, linked to kubeflow.